src/client.ts

```diff
@@ -267,25 +267,53 @@ export class GenesysCloudWebrtcSdk extends (EventEmitter as { new(): StrictEvent
         this.logger.info('window.beforeunload was called', { activeConversationsForClient: this.sessionManager.getAllActiveConversations() });
       });
 
-      /* if we are allowing softphone calls, we need station information */
-      if (this._config.allowedSessionTypes.includes(SessionTypes.softphone) && !this.isGuest) {
-        this.logger.info('SDK initialized to handle Softphone session. Requesting station');
-        const stationReq = this.fetchUsersStation()
-          .catch((err) => {
-            // these errors shouldn't halt initialization
-            this.logger.warn('error fetching users station', err);
-          });
-        const stationSub = this.listenForStationEvents();
-
-        await Promise.all([stationReq, stationSub]);
-      }
+      const sessionsToActivate = this._config.allowedSessionTypes;
+      this._config.allowedSessionTypes = [];
```

Comment:
This is kind of weird, but I didn't want to create another property to track which sessionTypes were active. I needed to clear the sessionTypes from the config because otherwise it gets eaten in the `addAllowedSessionType()` fn. I also didn't want to add some param like `isInitialAdd` or something like that to `addAllowedSessionType()` since it's a public api on the sdk.